package.json

```diff
@@ -68,50 +81,106 @@
     "jest-serializer-html": "^7.1.0",
     "jest-watch-typeahead": "^2.0.0",
     "nyc": "^15.1.0",
-    "playwright": "^1.14.0"
+    "playwright": "^1.14.0",
+    "playwright-core": "^1.14.0",
+    "semver": "^7.6.2"
   },
   "devDependencies": {
     "@auto-it/released": "^11.1.6",
+    "@aw-web-design/x-default-browser": "1.4.126",
     "@babel/cli": "^7.12.1",
     "@babel/preset-env": "^7.19.4",
     "@babel/preset-react": "^7.18.6",
     "@babel/preset-typescript": "^7.18.6",
     "@storybook/addon-a11y": "next",
     "@storybook/addon-coverage": "^1.0.0",
     "@storybook/addon-docs": "next",
+    "@storybook/docs-mdx": "4.0.0-next.1",
+    "@storybook/icons": "^1.4.0",
     "@storybook/react-vite": "next",
+    "@testing-library/dom": "10.4.0",
+    "@testing-library/react": "^14.0.0",
+    "@types/cross-spawn": "^6.0.6",
+    "@types/detect-port": "^1.3.0",
+    "@types/diff": "^5.0.9",
+    "@types/ejs": "^3.1.1",
+    "@types/find-cache-dir": "^5.0.0",
     "@types/jest": "^29.0.0",
+    "@types/js-yaml": "^4.0.5",
     "@types/node": "^16.4.1",
     "@types/node-fetch": "^2.6.11",
+    "@types/npmlog": "^7.0.0",
+    "@types/picomatch": "^2.3.0",
+    "@types/prettier": "^3.0.0",
+    "@types/pretty-hrtime": "^1.0.0",
+    "@types/semver": "^7.5.8",
     "@vitejs/plugin-react": "^4.0.3",
+    "@yarnpkg/esbuild-plugin-pnp": "^3.0.0-rc.10",
+    "@yarnpkg/fslib": "2.10.3",
+    "@yarnpkg/libzip": "2.3.0",
     "auto": "^11.1.6",
     "babel-jest": "^29.0.0",
     "babel-loader": "^8.1.0",
     "babel-plugin-istanbul": "^6.1.1",
+    "browser-dtector": "^3.4.0",
+    "bundle-require": "^5.1.0",
+    "camelcase": "^8.0.0",
     "can-bind-to-host": "^1.1.1",
+    "chai": "^5.1.1",
     "commander": "^9.0.0",
+    "comment-parser": "^1.4.1",
     "concurrently": "^7.0.0",
+    "cross-spawn": "^7.0.6",
+    "diff": "^5.2.0",
+    "es-toolkit": "^1.36.0",
+    "esbuild": "^0.18.0 || ^0.19.0 || ^0.20.0 || ^0.21.0 || ^0.22.0 || ^0.23.0 || ^0.24.0 || ^0.25.0",
+    "find-up": "^5.0.0",
     "glob": "^10.2.2",
+    "globby": "^14.0.1",
     "husky": "^8.0.0",
     "jest-image-snapshot": "^6.2.0",
+    "js-yaml": "^4.1.0",
+    "leven": "^4.0.0",
     "lint-staged": "^13.0.3",
+    "mlly": "^1.7.4",
+    "nanoid": "^4.0.2",
     "node-fetch": "^2",
+    "npmlog": "^7.0.0",
+    "open": "^8.4.0",
+    "p-limit": "^6.2.0",
+    "package-manager-detector": "^1.1.0",
+    "picocolors": "^1.1.0",
+    "picomatch": "^2.3.0",
+    "picoquery": "^1.4.0",
     "pkg-up": "^5.0.0",
     "prettier": "^2.8.1",
+    "pretty-hrtime": "^1.0.3",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
     "read-pkg-up": "^7.0.1",
-    "storybook": "next",
+    "sirv": "^2.0.4",
+    "slash": "^3.0.0",
+    "storybook": "^0.0.0-0 || ^8.2.0 || ^9.0.0 || ^9.1.0-0",
+    "storybook-src": "git+https://github.com/storybookjs/storybook.git#next",
+    "strip-ansi": "^7.1.0",
```

Comment:
There are a lot of devDependencies being added here, and they don't all seem to be used.  For example, searching for `strip-ansi` in this PR returns no results.